build/gen_stub.php

```diff
@@ -3686,6 +3741,30 @@ function generateFunctionEntries(?Name $className, array $funcInfos): string {
     return $code;
 }
 
+function generateAttributeInitialization(FileInfo $fileInfo, array &$classEntriesWithSensitiveParams): string {
+    $code = "";
+
+    foreach ($fileInfo->getAllFuncInfos() as $funcInfo) {
+        foreach ($funcInfo->args as $index => $arg) {
+            if (!$arg->isSensitive) {
+                continue;
+            }
+
+            if ($funcInfo->name instanceof MethodName) {
+                $classEntry = "class_entry_" . str_replace("\\", "_", $funcInfo->name->className->toString());
+                $functionTable = "&{$classEntry}->function_table";
+                $classEntriesWithSensitiveParams[] = $classEntry;
+            } else {
+                $functionTable = "CG(function_table)";
+            }
+
+            $code .= "    zend_mark_function_parameter_as_sensitive($functionTable, \"" . $funcInfo->name->getAttributeName() . "\", $index);\n";
```

Comment:
This needs to take conditional compilation of stubs into account. Example: ZipArchive::setEncryptionName().